MultiLabel/Core/src/main/java/org/tribuo/multilabel/evaluation/MultiLabelConfusionMatrix.java

```diff
@@ -162,9 +167,32 @@ public double confusion(MultiLabel predicted, MultiLabel truth) {
         return total;
     }
 
+    /**
+     * Sets the label order used in {@link #toString}.
+     *
+     * @param labelOrder The label order to use.
+     */
+    @Override
+    public void setLabelOrder(List<MultiLabel> labelOrder) {
+        if (labelOrder == null || labelOrder.isEmpty()) {
```

Comment:
Same question as before - can we easily check size here as well?

Reply:
Same answer, it's intentional to allow subsetting, but not documented or tested so I'll do that.